.github/workflows/bypass-version-review.yaml

```diff
@@ -0,0 +1,34 @@
+name: Bypass Version File Review by Admin
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  bypass-version-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up GitHub CLI Authentication
+        env:
+          GH_TOKEN: ${{ secrets.ADMIN_PAT }}
+        run: |
+          AUTHOR=$(gh pr view ${{ github.event.pull_request.number }} --json author --jq '.author.login')
+          ADMIN_CHECK=$(gh api user/memberships/orgs/scidsg --jq '.role')
+          if [ "$ADMIN_CHECK" != "admin" ]; then
+            echo "User is not an admin or is not a public member of the organization. Exiting."
+            exit 1
+          fi
+
+      - name: Check for hushline/version.py and approve if conditions are met
+        run: |
+          if git diff --name-only main ${{ github.head_ref }} | grep -q '^hushline/version.py$'; then
```

Comment:
Following the same suggestion as above,
```suggestion
          if ! git diff --name-only main ${{ github.head_ref }} | grep -v -q '^hushline/version.py$'; then
```

Reply:
Same as above...

![Screenshot 2024-09-03 at 9 56 10 AM](https://github.com/user-attachments/assets/d32efc69-8f3e-44de-b48a-4672aa77f568)

Reply:
Updated with non-failing exit